deps.ts

```diff
@@ -1,8 +1,7 @@
-export * as dotenv from "https://deno.land/std@0.188.0/dotenv/mod.ts";
-export * as discord from "https://deno.land/x/discord_api_types@0.37.42/v10.ts";
-export { Ngrok } from "https://deno.land/x/ngrok@4.0.1/mod.ts";
-export type { GitHubCodemodBuilderCreatePROptions } from "https://deno.land/x/codemod@v0.0.3/github/mod.ts";
-export { GitHubCodemodBuilder } from "https://deno.land/x/codemod@v0.0.3/github/mod.ts";
-export type { GitHubAPIClientOptions } from "https://deno.land/x/codemod@v0.0.3/github/api/mod.ts";
-export type { Officer } from "https://acmcsuf.com/code/raw/65d3cac9653f4469f51a4dad224d8878795ffc55/src/lib/public/board/types.ts";
+export * as dotenv from "https://deno.land/std@0.191.0/dotenv/mod.ts";
+export * as discord from "https://deno.land/x/discord_api_types@0.37.43/v10.ts";
+export * from "https://deno.land/x/codemod@0.0.5/github/mod.ts";
+export type { GitHubAPIClientOptions } from "https://deno.land/x/codemod@0.0.5/github/api/mod.ts";
+// TODO: Update SHA to main permalink blocked by <https://acmcsuf.com/pull/870>. Try manual testing.
```

Comment:
TODO(@EthanThatOneKid): Manual testing. Blocked by https://github.com/ethanthatonekid/acmcsuf.com/pull/870.

Reply:
Resolved in <https://github.com/acmcsufoss/boardd/pull/6/commits/f12bd7ffdd4c1fe10346e9011db3a503776b59a8>.